app/controllers/inventory_food_controller.rb

```diff
@@ -0,0 +1,32 @@
+class InventoryFoodController < ApplicationController
+  def new
+    @foods_available = Food.all
+  end
+
+  def create
+    new_inventory_food = InventoryFood.new(inventory_params)
+    if new_inventory_food.save
+      redirect_to "/inventory/#{params[:inventory_id]}", notice: 'Inventory was successfully created.'
+    else
+      render :new, alert: 'Error occurred, please try again. Inventory food not saved'
+    end
+  end
+
+  def destroy
+    inventory_food_to_delete = InventoryFood.find(params[:id])
+    if inventory_food_to_delete.destroy
+      redirect_to inventory_path(params[:inventory_id]), notice: 'Inventory food was successfully deleted.'
+    else
+      redirect_to inventory_path(params[:inventory_id]),
+                  notice: "Error. Inventory food wasn't deleted, please try again!."
+    end
```

Comment:
Great Job Alick, It is a really nice implementation UI/UX :+1:  and also logic :+1:  Once I add a food to inventory. I couldn't delete it. it will show up as an error

Reply:
Kindly elaborate on the error. A screenshot probably. It worked fine on local host when I last tried